index.html

```diff
@@ -1777,7 +1777,10 @@ <h2 id="Collection">Backbone.Collection</h2>
     <p id="Collection-remove">
       <b class="header">remove</b><code>collection.remove(models, [options])</code>
       <br />
-      Remove a model (or an array of models) from the collection, and returns them.
+      Remove a model (or an array of models) from the collection, and returns
```

Comment:
"Remove" doesn't agree with "returns" here (not your fault but let's fix this to either "Removes... returns them" or "Remove ... return them").